libuserhelper.cpp

```diff
@@ -161,6 +162,10 @@ bool LibUserHelper::removeUserFromGroup(const QString &user, const QString &grou
 
 uint LibUserHelper::addUser(const QString &user, const QString& name, uint uid, const QString &home)
 {
+    if (name.contains(',') || name.contains(':')) {
+        qCWarning(lcSUM) << "Invalid user name, comma or colon is not allowed";
+        return false;
```

Comment:
This method's return type is uint but false is boolean. Doesn't seem right to me.

Reply:
Yeah, need to fix.